package.json

```diff
@@ -23,6 +23,7 @@
     "commander": "2.9.0",
     "deepcopy": "0.6.3",
     "express": "4.14.0",
+    "firebase-admin": "^4.0.3",
```

Comment:
not sure we want to depend on that module for every single user.

You can probably provide that module externally.